jme3-core/src/main/java/com/jme3/anim/AnimLayer.java

```diff
@@ -249,13 +252,13 @@ public void setLooping(boolean loop) {
      * @param appDeltaTimeInSeconds the amount application time to advance the
      *     current Action, in seconds
      */
-    void update(float appDeltaTimeInSeconds) {
+    void update(float appDeltaTimeInSeconds, float speed) {
```

Comment:
please rename the variable _speed_ to `globalSpeed` and add the parameter to the javadoc as below:

```java
@param globalSpeed the global speed applied to all layers.
```